components/formats-api/src/loci/formats/FormatTools.java

```diff
@@ -1430,9 +1430,20 @@ public static Length getPhysicalSizeY(Double value, String unit) {
   }
 
   public static Length getPhysicalSizeZ(Double value) {
+    return getPhysicalSizeZ(value, null);
+  }
+
+  public static Length getPhysicalSizeZ(Double value, String unit) {
```

Comment:
Could you add here and above javadoc that if `unit == nulll`, `MICROM` will be used?